rcsbsearchapi/search.py

```diff
@@ -987,11 +989,15 @@ def _extract_identifiers(query_json: Optional[Dict]) -> List[str]:
 
     def _make_params(self, start=0):
         "Generate GET parameters as a dict"
+        results_content_type_list = ["experimental"]
+        if self.computational is True:
+            results_content_type_list.insert(0, "computational")  # includes computational in results_content_type parameter
```

Comment:
Just a general suggestion: I would suggest using the `.append()` method for extending a list in Python rather than `.insert()`, since it's generally cleaner, more common practice, and the order of the list doesn't matter here.